_posts/2017-12-04-example-project.md

```diff
@@ -0,0 +1,469 @@
+
+```json
+{
+  "priority": 0,
+  "match_concepts": null,
+  "content_partner": [
+    "University of Otago"
+  ],
+  "display_content_partner": [
+    "University of Otago"
+  ],
+  "display_collection": [
+    "Otago University Research Heritage"
+  ],
+  "primary_collection": [
+    "Otago University Research Heritage"
+  ],
+  "collection": [
+    "Otago University Research Heritage"
+  ],
+  "copyright": [
+    "All rights reserved"
+  ],
+  "usage": [
+    "All rights reserved"
+  ],
+  "dc_rights": [
+    "http://digital.otago.ac.nz/terms.php"
+  ],
+  "rights_url": [
+    "http://digital.otago.ac.nz/terms.php"
+  ],
+  "title": [
+    "Gog and Magog, Stewart Island. 1880."
+  ],
+  "description": [
+    "Lower left (l.l.) in ink: W. Deverell Jany 1880; margin below image in pencil: Gog &amp; Magog Stewart Island."
+  ],
+  "date": [
+
+  ],
+  "display_date": [
+
+  ],
+  "contributor": [
+
+  ],
+  "publisher": [
+
+  ],
+  "subject": [
+    "Islands",
+    "Landscape"
+  ],
+  "source": [
+    "Found uncatalogued in Hocken 1947. Dr T.M. Hocken’s Collection"
+  ],
+  "creator": [
+    "unknown"
+  ],
+  "dc_type": [
+    "Image",
+    "Still Image",
+    "Watercolors",
+    "Art"
+  ],
+  "format": [
+
+  ],
+  "category": [
+    "Images"
+  ],
+  "landing_url": [
+    "http://otago.ourheritage.ac.nz/items/show/4480"
+  ],
+  "internal_identifier": [
+    "http://otago.ourheritage.ac.nz/items/show/4480"
+  ],
+  "large_thumbnail_url": [
+    "http://s3.amazonaws.com/ourheritagemedia%2Ffullsize%2Fa24297b36e4d17d4b8bd4da8f7d6bb6c.jpg"
+  ],
+  "thumbnail_url": [
+    "http://s3.amazonaws.com/ourheritagemedia%2Fsquare_thumbnails%2Fa24297b36e4d17d4b8bd4da8f7d6bb6c.jpg"
+  ],
+  "dc_identifier": [
+    "oai:otago.ourheritage.ac.nz:4480"
+  ],
+  "source_id": "otago-hocken",
+  "data_type": "record"
+}
+```
+
+If you have a successful preview, close the modal window, scroll to the bottom of the page, enter some text in the message field, and click Update Parser Script.  Next, in the righthand columns, click on the name of your parser script under **History**, then click the blue button **Tag as Staging**.
+
+You can now run a staging harvest, note that ‘Staging’ is a term bound to the Manager and does not relate to the Rails environment.
+
+Click ‘Run Harvest’, and then click ‘Staging Harvest’, enter 1000, and then click start. I do not know how large this example harvest is.
+
+### Step Seven - Indexing your Records
+
+Once the harvest has finished you are now ready to index your records into Solr. On our DNZ servers we have a background job that handles this, but since we are running locally we can just do it in the Rails console of the API.
+
+Cd into your api repository, enter the rails console and run the following.
+
+```Ruby
+Sunspot.session = Sunspot::Rails.build_session
+SupplejackApi::Record.all.map(&:index)
```

Comment:
You can run `Sunspot.index(Record.all)` which will only perform 1 call to Solr.

Reply:
Cool I will update!